src/portable/profile/trusted/service.conf

```diff
@@ -2,6 +2,5 @@
 
 [Service]
 MountAPIVFS=yes
-BindPaths=/run
```

Comment:
@poettering this changes the trusted profile semantics though, doesn't it? Before this change, /run from the host was overmounted. Now it will be an empty tmpfs (with the additional bits we add).

Reply:
Yeah I fucked that up: #18336.